internal/integration_test/vs/bench_memory.go

```diff
@@ -14,6 +14,7 @@ const (
 	i32ValueMemoryOffset = 32
 	i64                  = "i64"
 	i64ValueMemoryOffset = 64
+	inWasmIteration      = 100
```

Comment:
setting this to >=1000, we can see the actual diff, and I found that there's room for optimization in our memory access code